README.md

```diff
@@ -108,145 +40,59 @@ To proceed, click on the **Create a Bot** button and fill up all the required in
+#### 2. Open Terminal and Define Working Directory
 
-* **Windows Users**
+Open a terminal and you can start working on your home directory (`/Users/<username>` for macOS, `<root>\Users\<username>` for Windows or `/home/<username>` for Linux). Otherwise, you can navigate to another directory using `cd <other-directory>`.
 
-Open the **Docker Quickstart Terminal** from the **Start Menu**. Once you see an ASCII boat in the terminal (the Docker logo) and can type in the terminal, navigate to the **Public** directory by running the following command.
+#### 3. Clone git Repository and Install Dependencies
 
-````sh
-cd /c/Users/Public
-````
-
-Note that if `c` isn't your Windows drive letter, you must replace it with the correct drive letter.
-
-Remember that you must continue working on this terminal for the following steps.
-
-### 3. Clone git Repository
-
-Clone the current git repository recursively with submodules to your local machine by running the following command on your terminal.
+Clone the git repository to your local machine by running the following command on your terminal.
 
 ```sh
-git clone --recursive https://github.com/haskalpa/docker-webexteams-bot-example.git
+git clone https://github.com/sarupanda/webexteams-python-example.git
 ```
 
-If you run the command `ls`, you will find all the directories and files inside your current directory. To go to directory you have just cloned simply run `cd docker-webexteams-bot-example`. Try running `ls` and check that you can see all the files you will need to get your Bot up and running.
-
-### 4. Configure with Access Token
-
-Remember the access token for the bot that you saved earlier? Here is where you will need it.
-
-Open the folder that you have just created, **docker-webexteams-bot-example**. Inside you will find a folder named **config**, open it. Then, create a copy of the **config_example.yaml** file and rename it to **config.yaml**.
-
-Open the new file with your favourite text editor. If you still do not have one, take a quick look at [Atom](https://atom.io/), [Visual Studio Code](https://code.visualstudio.com/) or (in case you are a very brave developer) [Vim](https://www.vim.org). This is what the file contents should look like:
+To go to directory you have just cloned simply run `cd webexteams-python-example`. Try running `ls` and check that you can see all the files you will need to get your Bot up and running.
 
-````sh
----
-hello_bot:
-teams_access_token: <my_bot_access_token>
-````
-
-Yep, you guessed it! Replace `<my_bot_access_token>` with the Access Token you save during the **Create a Bot** step.
-
-### 5. Build Docker container
-
-Back to the terminal window: run the following command to build your Docker container.
+After installing Python, open terminal and run the following commands to install dependencies.
```

Comment:
Command*, because thank god we removed Docker from this 😃

Reply:
thanks for noticing this!